test/e2e/pkg/status_resync_test.go

```diff
@@ -171,8 +171,10 @@ var _ = Describe("Status Resync After Restart", Ordered, Label("e2e-tests-status
 				return fmt.Errorf("nginx deployment still exists")
 			}, 2*time.Minute, 2*time.Second).ShouldNot(HaveOccurred())
 
-			err = agentTestOpts.kubeClientSet.CoreV1().ServiceAccounts("default").Delete(ctx, name, metav1.DeleteOptions{})
-			Expect(err).ShouldNot(HaveOccurred())
+			// In a ROSA cluster the sa in default namespace is protected, the sa cannot be deleted directly
+			// see https://access.redhat.com/solutions/7055600
+			// err = agentTestOpts.kubeClientSet.CoreV1().ServiceAccounts("default").Delete(ctx, name, metav1.DeleteOptions{})
```

Comment:
Do we create the `default` serviceaccount? I think it should be there by default.
@morvencao comments?

Reply:
we assume `default` sa is there, it's not created by us.
the code change looks good to me, we can remove this.

Reply:
Thanks @morvencao 
@skeeey Let us remove it directly

Reply:
removed